app/composables/Date.ts

```
@@ -1,7 +1,7 @@
 // International Date Detection
 export function formatDate(string: string) {
   const { locale } = useI18n();
-  const options: Object = { year: "numeric", month: "long", day: "numeric" };
+  const options: Object = { day: "numeric", month: "short", year: "numeric" };
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)